src/_P147_SGP4x.ino

```diff
@@ -0,0 +1,236 @@
+#include "_Plugin_Helper.h"
+#ifdef USES_P147
+
+// #######################################################################################################
+// ############################ Plugin 147: Gases - SGP4x CO2 (VOC), NOx (SGP41) #########################
+// #######################################################################################################
+
+/**
+ * 2023-04-29 tonhuisman: Implement sensor raw reading
+ * 2023-04-27 tonhuisman: Start of plugin
+ *
+ * Using direct I2C functions and Sensirion VOC/NOx calculation library
+ **/
+# define PLUGIN_147
+# define PLUGIN_ID_147          147
+# define PLUGIN_NAME_147        "Gases - CO2 SGP4x"
```

Comment:
I suggest to change "Gases - CO2 SGP4x" to "Gases - VOC SGP4x" since SGP4x are VOC (volatile organic compounds) sensors and not CO2 sensors.

Reply:
or, even better, name it as "Gases - SGP4x" since currently the name for SGP30 (already in ESPEasy) is "Gasses - SGP30"

Reply:
Then I would suggest to change the SGP30 one into having "CO2" in the name.

Reply:
and no... SGP30 like SGP4x are VOC sensors, not CO2 sensors.
So if SGP4x will be renamed as "Gases - SGP4x", SGP30 should remain as is it: "Gasses - SGP30".

Reply:
Check, VOC sensors should never be named CO2.

Point is that the user now has to know the exact type beforehand, which does make sense as you are trying to wire up something that's probably in front of you.
But when looking for an alternative (which is not uncommon if you try to wire up a VOC sensor at first ;) ) then it would be great to have a quick look at the other supported sensors and immediately see what you might consider as an alternative.